dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -129,7 +129,9 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
 
     inline wil::com_ptr<INotificationsLongRunningPlatform> GetNotificationPlatform()
     {
-        return wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER);
+        wil::com_ptr<INotificationsLongRunningPlatform> platform;
+        THROW_IF_FAILED(CoCreateInstance(CLSID_NotificationsLongRunningPlatform, nullptr, (CLSCTX_LOCAL_SERVER | CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION), IID_PPV_ARGS(&platform)));
```

Comment:
Warning: CoCreateInstance rejects unknown CLSCTX flags, and CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION will not be available on all machines that support WinAppSdk (the feature is serviced to all RS5+ releases, but you can't always assume that the machine you're running on is fully patched).

Be sure to consider whether the failure mode of passing this flag on an unpatched system is what you want, or if you need to do runtime probing for whether CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION is supported to get correct behavior on those systems.